content/applications/sales/sales/sales_quotations.rst

```diff
@@ -0,0 +1,138 @@
+:show-content:
+:hide-page-toc:
+
+================
+Sales quotations
+================
+
+A *sales quotation* or quote is a document sent to a customer that outlines the estimated costs and
+terms for goods and services. Once accepted, a quotation can be converted into a sales order, which
+serves as the final agreement before delivery and invoicing.
+
+Sales flow overview
+===================
+
+Quotations fit into a broader sales flow that connects different stages of customer interactions
+from initial interest to payment.
+
+The typical flow follows these steps:
```

Comment:
This is a good start but is only partially true.

It depends on the business type (b2b, b2c) and if integrations like ecom, pos, etc. are integrated.

In eCom, for example, the quotation is instantly created and confirmed once payment is captured...the invoice comes later and delivery is last.